LICENSE

```diff
@@ -2378,3 +2378,28 @@ The externally maintained libraries used by Node.js are:
     OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
      ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   """
+
+- on-exit-leak-free, located at lib/internal/process/finalization, is licensed as follows:
```

Comment:
This needs to be added to license builder as well

Reply:
Hm... in this case, remove from LICENSE and leave only the license-builder, or keep both?

Reply:
I kept both.